src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -4790,6 +4790,45 @@ namespace OutputReportTabular {
         // The routine generates an annual table with the following columns which correspond to
         // the output variables and data structures shown.
 
+        // Using/Aliasing
```

Comment:
So the Gather function is still called from wherever it was previously called from.  This simply extracts out a portion of it into a separate Calc function which is called from the HVACManager, as well as from inside here.  OK.